### ISSUE TITLE
Restarts on meshes refined more than once

### ISSUE THREAD

**Issue by vikramvgarg** [grinsfem/grins#293]
I have mesh 1 called mesh_1.xda and a solution solution_on_mesh_1.xda. For my next simulation I want to read in mesh_1.xda, uniformly refine it and use solution_on_mesh_1.xda as the restart solution.

When I do only one uniform refinement (uniformly_refine = '1') the simulation runs fine.

However, when I do two uniform refinements (uniformly_refine = '2') I get the assertion failure:
Restarting from /workspace/vikram/grins_libmesh_master_dbg/examples/sa_airfoil/turbulent_airfoil_Re_200000_aoa_16_deg.xda
Assertion `!elem->has_dofs(sys_num) || elem->p_refinement_flag() == Elem::JUST_REFINED || elem->old_dof_object' failed.

If I understand what your trying to say, GRINS can only project solutions on meshes one uniform refinement away from the one the restart solution is on.

**Comment by nicholasmalaya** [comment]
> If I understand what your trying to say, GRINS can only project solutions on meshes one uniform refinement away from the one the restart solution is on.

That is correct. Only one refinement is possible upon restarts at present.

**Comment by vikramvgarg** [comment]
Right, thats the issue. Sorry I copied the text from an email to Paul.

**Comment by nicholasmalaya** [comment]
Is this a feature request then?

**Comment by pbauman** [comment]
This seems like a bug to me. Bug/feature request/whatever, it should get fixed (at some point...). I've hit this snag too.
